src/commands/config/index.ts

```diff
@@ -93,6 +116,13 @@ class ConfigCommand extends Command {
                   .setRequired(true)
                   .addChoices(...RANDOM_VOICE_CHANNEL_NAMES_FREQUENCY_CHOICES)
               )
+              .addStringOption((option) =>
+                option
+                  .setName('timezone')
+                  .setDescription(
+                    'Which timezone it will update relative to. (Default: UTC)'
+                  )
+              )
```

Comment:
How do we feel about this being an optional param for both of the commands? I'm perfectly torn between this and having timezone be its own subcommand. Thoughts?

Reply:
I think having it as an option is cleaner personally